packages/rpc-subscriptions/README.md

```diff
@@ -18,6 +18,16 @@ This package contains types that implement RPC subscriptions as required by the
 
 ## Functions
 
+### `getChannelPoolingChannelCreator({ createChannel, maxSubscriptionsPerChannel, minChannels })`
```

Comment:
Since this is essentially a channel creator decorator, how would you feel about making the `createChannel` attribute a proper argument so it can be piped more easily?

Something like:

```ts
const channelCreator = pipe(
  getMyOriginalChannelCreator(),
  cc => getChannelPoolingChannelCreator(cc, { ... }),
)
```